crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -300,19 +300,7 @@ impl CallContractCommand {
 		};
 
 		// Resolve message arguments.
-		let mut contract_args = Vec::new();
-		for arg in &message.args {
-			let mut input = cli
-				.input(format!("Enter the value for the parameter: {}", arg.label))
-				.placeholder(&format!("Type required: {}", arg.type_name));
-
-			// Set default input only if the parameter type is `Option` (Not mandatory)
-			if arg.type_name.starts_with("Option<") {
-				input = input.default_input("");
-			}
-			contract_args.push(input.interact()?);
-		}
-		self.args = contract_args;
+		self.args = request_contract_function_args(&message, cli)?;
```

Comment:
Small clippy warning

Reply:
Shouldn't this fail in the CI? I am actually seeing many things that could be autofixed with clippy 🤔  that are not yelling in the CI.

Reply:
Should be fixed in [d31d86f](https://github.com/r0gue-io/pop-cli/pull/606/commits/d31d86f806ed1ed7337f2a078ed302ae36f1d71a).

Reply:
In my opinion, yes. Otherwise we leave clippy issues around and never bother to fix them.